molgenis_documentation/user_documentation.md

```diff
@@ -494,7 +494,51 @@ In the following paragraphs we explain on how to use different modules to config
 The home page is the first thing a visitor sees. It should be the thing that informs a user about whatever is on the page he or she has navigated to. It should also sell the content, making it a very important piece
 
 #### <a name="user-management"></a> User management
-User management
+User management is crucial for keeping an overview of people visiting your research database, but it is also important for security reasons. MOLGENIS has an extensive user management system, allowing people to register themselves, or be registered by an administrator. You can find the User manager module under the Admin menu. Going there, you will find something looking like this:
+
+![Menu manager screen](images/user_manager.png?raw=true, "user manager")
+
+The user management menu allows you to create new users and groups. But also lets you edit existing users, or add users to certain groups. The users *admin* and *anonymous* always exist. The admin user, as the name suggests, is the administrator user. The anonymous user is used for people navigating to your website, but are not logged in. This means that giving rights to the anonymous user will give permissions for everyone, also those that are not registered in your system. Note that it is not possible to delete users! You can only set them to inactive, which will prevent him or her from logging in.
+
+
+**Try it out**  
+To let you get a feel of how the user manager works, we will create a new user called molgenis_user. First, click the pencil with green plus icon. This will open up a form for creating a new MolgenisUser. Most of the fields are pretty self explanatory, but there are a few that we will elaborate on. 
```

Comment:
button: I already uploaded to the images directory new.png, will put up pull request